package.json

```diff
@@ -74,9 +74,9 @@
     "pivottable": "^2.15.0",
     "plotly.js": "1.41.3",
     "prop-types": "^15.6.1",
-    "react": "^16.3.2",
+    "react": "^16.7.0",
```

Comment:
Interesting - what did you need this for?

Reply:
Ooops... I needed it for react contexts. Will revert

Reply:
...or maybe keep?

Reply:
Good question. No real answer 😅

Reply:
I decided to revert because it's not really related to this PR.